packages/patternfly-4/react-core/src/components/ApplicationLauncher/ApplicationLauncherMenu.tsx

```diff
@@ -65,7 +63,7 @@ class ApplicationLauncherMenu extends React.Component {
     }
   }
 
-  sendRef = (index, node, isDisabled) => {
+  sendRef = (index: number, node: any, isDisabled: boolean) => {
```

Comment:
should node be type any?

Reply:
We call it like this `this.props.sendRef(this.props.index, this.ref.current);` and the type of this.ref.current is any so I think it should be type any.